src/app/utils/item.js

```diff
@@ -1,120 +1,186 @@
 import Locations from '../../../locations.js';
 import LocationCodes from '../../../locationCodes.js';
 import {
+  findWhere as _findWhere,
   isEmpty as _isEmpty,
+  extend as _extend,
 } from 'underscore';
 
 function LibraryItem() {
+  /**
+   * getDefaultLocation()
+   * Return the default delivery location for an item.
+   * @return {object}
+   */
   this.getDefaultLocation = () => ({
     '@id': 'loc:mal',
     prefLabel: 'Stephen A. Schwarzman Building - Rose Main Reading Room 315',
+    customerCode: 'NH',
   });
 
   /**
-   * getItem(record, itemId)
-   *
-   * @param {Object} record
-   * @param {String} itemId
-   * @return {Object}
+   * defaultDeliveryLocations()
+   * Temporarily return three hardcoded and default delivery locations.
+   * @return {array}
    */
-  this.getItem = (record, itemId) => {
-    let thisItem = {};
-    // look for item id in record's items
-    const items = (record && record.items) ? record.items : null;
-
-    if (items && itemId) {
-      items.forEach((i) => {
-        if (i['@id'] && i['@id'].substring(4) === itemId) {
-          thisItem = i;
-        }
-      });
+  this.defaultDeliveryLocations = () => [
+    {
```

Comment:
So this is hardcoded now and will be removed or update based on the needs in the future. Is that correct?